src/live-programming.rst

```diff
@@ -26,6 +26,64 @@ Interactive Debuggers
 Program Visualization
 ---------------------
 
+**Program visualization** is the use of various techniques to enhance the human understanding of computer programs from distinct aspects including the structure of the source code and run-time behavior.
+
+There are **program visualization** systems designed to aid significantly the practice of professional programming and software engineering as well as the learning of computer science by novice programmers. And it turns out that **program visualization** doesn't have to deal with toy programs, e.g., `Frappe - Visualizing Code Dependency Graphs <https://softvis.wordpress.com/2018/08/31/frappe-visualizing-code-dependency-graphs/>`_.
+
+.. container:: bib-item
+
+  .. bibliography:: live-programming.bib
+    :filter: key == "price1993principled"
+
+ Note that **program visualization** is different from **software visualization** in that the latter includes the visualization of a high-level description of algorithm or a piece of software which is in contrast to **program visualization** where actual implemented code is visualized.
+
+.. container:: bib-item
+
+  .. bibliography:: live-programming.bib
+    :filter: key == "maletic2002task"
+
+ This paper defines five dimensions, Tasks, Audience, Target, Representation and Medium to describe several software visualization systems that have very different features along those defined dimensions. By realigning taxonomies with the perspective of current software engineering problems, open research issues are identified. Additionally, the paper identifies the strengths of individual tools and the techniques they apply.
```

Comment:
* Might be worthwhile to bullet point the five dimensions and briefly describe what they mean here.
* What does "realigning taxonomies" mean? Consider rephrasing more clearly / not using jargon